content/docs/tinacms-context.md

```diff
@@ -0,0 +1,182 @@
+---
+title: Next.js APIs
+id: '/docs/tinacms-context'
+---
+
+There are essentially two touch points you'll need set up in order for Tina to work with your Next.js application or website.
+
+1. Load data from the TinaCMS GraphQL API from `getStaticProps`
+2. Conditionally wrap your app in the TinaCMS context
+
+## `getStaticPropsForTina`
+
+TinaCMS is easiest to work with when you provide a predictable shape to the props you return from `getStaticProps`. `getStaticPropsForTina` enforces this shape for you:
+
+```tsx
+// pages/home.js
+import { getStaticPropsForTina } from 'tinacms'
+
+const getStaticProps = () => {
+  return getStaticPropsForTina({
```

Comment:
I thought we wanted this API to look something like this.

```ts

export const getStaticProps = () => {
  const tinaProps = getStaticPropsForTina({
    query: `
      query GetPostDocument($relativePath: String!) {
        getPostDocument(relativePath: $relativePath) {
          title
          body
        }
      }
    `,
    variables: {
      relativePath: 'hello-world.md',
    },
  })
return {
props: {
...tinaProps,
myOtherProp: "test",
},
}
```

Reply:
Thanks @logan-anderson, you're right that's the actual API